script.artistslideshow/.gitignore

```diff
@@ -0,0 +1,130 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
please drop from the submission

Reply:
I'll have to manually submit this rather than using the tool I found for submitting from my master branch, as I can't find a way for GitHub to ignore my gitignore file (even if I add .gitignore to the ignore file).  This is apparently a "feature" of github know so that people sharing repos can share gitignore files.  But once the other comments on this are resolved I'll close this PR and resubmit without this file.